log_queries.py

```diff
@@ -0,0 +1,78 @@
+import datetime
+
+import pickle
+import os.path
+
+from googleapiclient.discovery import build
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.transport.requests import Request
+
+# If modifying these scopes, delete the file token.pickle.
+SCOPES = ['https://www.googleapis.com/auth/spreadsheets']  # Allows read and write access
+
+# The ID, range, and auth path for appending to the spreadsheet.
+SPREADSHEET_ID = '15f_zVJxNnUz_9G5ZYPery68rR8PyUnO0yWKSyGQjy4w'
```

Comment:
Our sheet is publicly accessible to anyone with the link, so putting the ID here gives anyone who wants access the info they need to get in! I don't think there's much someone could gain from looking at it, but this isn't great practice, and a malicious actor could wipe our logs. We could either switch to restricting access to specific accounts, or load the `SPREADSHEET_ID` from either an environment variable or directly from a file. I think simply restricting the permissions on the google sheet is fine for now. 

We dealt with a similar problem in the nimbus api by [creating a file with a folder ID from an environment variable](https://github.com/calpoly-csai/api/blob/947ed368ffa2f06da9b60618504e4be660ced602/setup_special_files_from_env.py#L161-L165), then [reading in that file](https://github.com/calpoly-csai/api/blob/947ed368ffa2f06da9b60618504e4be660ced602/flask_api.py#L749); this is probably more complicated than what we need here, but is a potential solution.

Reply:
I agree that restricting permissions should be fine.

Reply:
Ok so you're fine with me keeping the ID in for now?

Reply:
I think keeping the ID in makes sense right now - just make the document only editable by specific users.